lib/src/transaction.dart

```diff
@@ -126,4 +127,41 @@ class FinalizedTransaction {
 			return client.call(base._keys, raw, _function, chainId: chainId);
 		});
 	}
+}
+
+class TransactionInformation {
+	String blockHash;
```

Comment:
Let's add some documentation to indicate that this field can be null when this transaction has not been included in a block:
```suggestion
	/// The hash of the block containing this transaction. If this transaction has not been mined yet and is thus in no
	/// block, it will be `null`
	String blockHash;
```